.editorconfig

```diff
@@ -0,0 +1,25 @@
+# https://editorconfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.{java,kt,kts,scala,rs,xml,kt.spec,kts.spec}]
+indent_size = 4
+
+[*.{kt,kts}]
+ij_kotlin_imports_layout=*
+ij_kotlin_allow_trailing_comma=true
```

Comment:
I saw that `super-linter` is failing on some files due to trailing commas. I'm not sure how to modify the rules that `super-linter` follows, or how to make sure it stays up to date with the latest version of ktlint. I recommend replacing that with something you can integrate and run locally, and that allows custom rules (like ktlint does with .editorconfig).

There are a few different gradle plugins for projects out there that do more than just check Kotlin code if you want something more than what ktlint provides on its own.